GGSmtp/GGSmtp.Tests/DataResponderTests.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using NUnit.Framework;
+using Simple.MailServer.Smtp.Config;
+using Moq;
+using Simple.MailServer.Smtp;
+using System.IO;
+using Simple.MailServer;
+using System.Text;
+
+namespace GGSmtp.Tests
+{
+    [TestFixture]
+    public class DataResponderTests
+    {
+        [Test(Author = "Dor", Description = "Checks the DefaultDataResponder", TestOf = typeof(DefaultDataResponder)]
+        public void TestAttachmentSend()
```

Comment:
look at (or talk wit me about) our test-methods naming and arrangement conventions !fix